examples/hacker_news_assets/hacker_news_assets/activity_analytics/__init__.py

```diff
@@ -0,0 +1,68 @@
+import json
+import os
+
+from dagster_dbt import dbt_cli_resource
+from dagster_dbt.asset_defs import load_assets_from_dbt_manifest
+from hacker_news_assets.resources import RESOURCES_LOCAL, RESOURCES_PROD, RESOURCES_STAGING
+from hacker_news_assets.sensors.hn_tables_updated_sensor import make_hn_tables_updated_sensor
+
+from dagster import AssetGroup
+from dagster.utils import file_relative_path
+
+from . import assets
+
+DBT_PROJECT_DIR = file_relative_path(__file__, "hacker_news_dbt")
+DBT_PROFILES_DIR = DBT_PROJECT_DIR + "/config"
+dbt_staging_resource = dbt_cli_resource.configured(
+    {"profiles-dir": DBT_PROFILES_DIR, "project-dir": DBT_PROJECT_DIR, "target": "staging"}
+)
+dbt_prod_resource = dbt_cli_resource.configured(
+    {"profiles_dir": DBT_PROFILES_DIR, "project_dir": DBT_PROJECT_DIR, "target": "prod"}
+)
+
+
+dbt_assets = load_assets_from_dbt_manifest(
+    json.load(open(os.path.join(DBT_PROJECT_DIR, "target", "manifest.json"), encoding="utf-8")),
+    io_manager_key="warehouse_io_manager",
+)
+
+activity_analytics_assets_prod = AssetGroup.from_package_module(
+    package_module=assets, resource_defs=RESOURCES_PROD
+).prefixed("activity_analytics") + AssetGroup(dbt_assets, resource_defs=RESOURCES_PROD)
+
+activity_analytics_assets_staging = AssetGroup.from_package_module(
+    package_module=assets, resource_defs=RESOURCES_STAGING
+).prefixed("activity_analytics") + AssetGroup(dbt_assets, resource_defs=RESOURCES_STAGING)
```

Comment:
I think this will be much less of an issue in "resources on a repository"-land, but this does feel like a ton of copy-paste. Any particular reason not to have this as generator function in the meantime (i.e. get_activity_analytics_assets(resources))?